### PR TITLE
Use new Vault helm chart in quickstart.

### PR DESCRIPTION
**Pull request by revit13**
Signed-off-by: Revital Sur <eres@il.ibm.com>

### CODE REVIEW

File: site/docs/get-started/quickstart.md [hunk 1]
```diff
@@ -32,35 +32,64 @@ helm install cert-manager jetstack/cert-manager \
...
 
     ```bash
     helm repo add hashicorp https://helm.releases.hashicorp.com
-    helm repo update
-    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
-        --set "server.dev.enabled=true" \
-        --set "injector.enabled=false" \
-        --values https://raw.githubusercontent.com/mesh-for-data/mesh-for-data/v0.1.0/third_party/vault/vault-single-cluster/values.yaml \
-        --wait --timeout 120s
+    helm dependency update m4d-charts/vault
```
Comment (shlomitk1) [inline site/docs/get-started/quickstart.md:74]: charts/vault?

File: site/docs/get-started/quickstart.md [hunk 2]
```diff
@@ -32,35 +32,64 @@ helm install cert-manager jetstack/cert-manager \
...
     ```
 
 === "OpenShift"
 
     ```bash
     helm repo add hashicorp https://helm.releases.hashicorp.com
-    helm repo update
-    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+    helm dependency update m4d-charts/vault
+    helm install vault m4d-charts/vault --create-namespace -n m4d-system \
```
Comment (shlomitk1) [inline site/docs/get-started/quickstart.md:87]: charts instead of m4d-charts?

**Review by shlomitk1 — CHANGES_REQUESTED** [review]
m4d-charts does not exist.
Installation using charts/vault seems ok.

**Comment by roee88** [comment]
> m4d-charts does not exist.
> Installation using charts/vault seems ok.

I didn't follow the changes so it might be unrelated to your comments, but just in case: m4d-charts is our published charts. The vault chart will be published there in the next release. The instructions here update the dev version of the docs and as such only the "install latest development version from GitHub" must work. The rest must work once the next release is created.

**Comment by shlomitk1** [comment]
> 
> 
> > m4d-charts does not exist.
> > Installation using charts/vault seems ok.
> 
> I didn't follow the changes so it might be unrelated to your comments, but just in case: m4d-charts is our published charts. The vault chart will be published there in the next release. The instructions here update the dev version of the docs and as such only the "install latest development version from GitHub" must work. The rest must work once the next release is created.

Ah, ok, this explains why 'helm install vault m4d-charts/vault' did not work in my testing :-)

**Review by shlomitk1 — APPROVED** [review]

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**